### DISCUSSION TITLE
Disabling screensaver/monitor shutdown

### DISCUSSION THREAD

**Discussion by MarcoPaun** [lxqt/lxqt#2426] [Q&A] [answered]
Hiall, I cannot disable screensaver/monitor shutdown. After a few minutes it gives me "can't load xdg-screensaver lock".

Can you give me a hint? I am trying this on LXQT, in case it it matters.

Thanks!

**Comment by luis-pereira** [comment, marked as answer]
Check this:
![screen-1](https://github.com/lxqt/lxqt/assets/62877/0331b2e7-0ac2-461f-94e1-21decba5fed8)
![screen](https://github.com/lxqt/lxqt/assets/62877/119d0a74-c601-406f-8cb9-ba68ad37339d)
